content/oci/oci_test.go

```diff
@@ -654,6 +672,15 @@ func TestStore_DisableAutoSaveIndex(t *testing.T) {
 	if _, err := os.Stat(s.indexPath); err != nil {
 		t.Errorf("error: %s does not exist", s.indexPath)
 	}
+
+	// test untag
+	err = s.Untag(ctx, ref)
+	if err != nil {
+		t.Fatal("Store.Untag() error =", err)
+	}
+	if got, want := len(internalResolver.Map()), 1; got != want {
```

Comment:
We should test like line 662-671

Reply:
I think we can tag the descriptor with 2 different references at the beginning, so we have 3 entries in the internal resolver:
```
"foo" -> desc
"bar" -> desc
desc.Digest -> desc
```
After manually calling `saveIndex`, we have the 2 entries in `index.Manifests`:
```
desc ["org.opencontainers.image.title": "foo"]
desc ["org.opencontainers.image.title": "bar"]
```

And then we untag "bar", the internal resolver map becomes:
```
"foo" -> desc
desc.Digest -> desc
```
While `index.Manifests` has no change because we disabled auto save.

After manually calling `saveIndex`, we have only 1 entry left in `index.Manifests`:
```
desc ["org.opencontainers.image.title": "foo"]
```